lib/libknc.c

```diff
@@ -2251,6 +2251,18 @@ fdwritev(void *cookie, const struct iovec *iov, int iovcnt)
 #ifdef O_NOSIGPIPE
 	return writev(fd, iov, iovcnt);
```

Comment:
We could just always use sendmsg(2) and reduce the amount of #ifdef mess by a teeny bit.